worlds/stardew_valley/data/locations.csv

```diff
@@ -1,2495 +1,2722 @@
-id,region,name,tags,mod_name

```

Comment:
I can fix this file if you want, but it'll have to get fixed regardless (line breaks)

Reply:
Yeah weird change but likely due to me building the shipping names.